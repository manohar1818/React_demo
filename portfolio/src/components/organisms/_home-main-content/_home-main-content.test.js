import React from 'react';
import { create } from "react-test-renderer";
import { render ,fireEvent} from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import HomeMainContent from './_home-main-content';

describe(" HomeMainContent component",() =>{
    test("Matches the snapshot",() => {
        const galleryAdmin= create(< HomeMainContent/>);
        expect(galleryAdmin.toJSON()).toMatchSnapshot();

    })
    test("GalleryAdmmin is rendered",() => {
        const { getByTestId } = render(< HomeMainContent/>);
        expect(getByTestId("home")).toBeInTheDocument();
        
    });
   
   
   

});