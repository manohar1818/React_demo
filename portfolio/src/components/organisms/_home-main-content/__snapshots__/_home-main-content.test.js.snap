// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` HomeMainContent component Matches the snapshot 1`] = `
<div
  className="makeStyles-styleHeaderElement-1"
  data-testid="home"
>
  <h2
    className="MuiTypography-root makeStyles-name-3 MuiTypography-h2"
  >
    Hii there,this is Ruchitha
  </h2>
  <h4
    className="MuiTypography-root MuiTypography-h4"
  >
    Full Stack Developer Intern
  </h4>
  <div
    className="makeStyles-picture-2"
  >
    <div
      className="makeStyles-styleImage-4"
      data-testid="image"
    >
      <div
        className="makeStyles-picture-5"
        style={
          Object {
            "backgroundImage": "URL(https://thumbs.dreamstime.com/b/young-woman-avatar-cartoon-character-profile-picture-young-brunette-woman-short-hair-avatar-cartoon-character-vector-149728784.jpg)",
          }
        }
      />
    </div>
  </div>
</div>
`;
